cornice/validators/_colander.py

```diff
@@ -62,8 +60,7 @@ def __new__(cls, name, bases, class_attrs):
                 class_attrs[location] = _ensure_instantiated(schema)
                 return type(name, bases, class_attrs)
 
-        class RequestSchema(with_metaclass(
-                RequestSchemaMeta, colander.MappingSchema)):
+        class RequestSchema(colander.MappingSchema, metaclass=RequestSchemaMeta):  # noqa
             """A schema to validate the request's location attributes."""
             pass
 
```

Comment:
```suggestion
        class RequestSchema(colander.MappingSchema):
            __metaclass__ = RequestSchemaMeta
            """A schema to validate the request's location attributes."""
            pass

```

Reply:
Python2 style?

Reply:
According to six doc it is python3 style :joy:

Reply:
Oh no you are right, I misread: https://six.readthedocs.io/#six.add_metaclass